### ISSUE TITLE
FAQ section not centered

### ISSUE THREAD

**Issue by jeffdomke** [blockstack/app.co#399]
![Screen Shot 2019-01-07 at 11:39 PM](http://marker.screenshots.prod.s3.amazonaws.com/c55c399014784b3a8a632cc2336c47e0-1546904395682.png)
---
### More Info

**Source URL**: [https://app.co/mining#learn-more](https://app.co/mining#learn-more)

**Comment by aulneau** [comment]
I think this could go either way. Nothing else is really centered, and it would be hard to have the table of contents centered, and then center the larger bodies of content. I prefer it this way, with shorter line widths and left justified. Can play around with it though.

**Comment by jeffdomke** [comment]
Think I mean the other sections of the page have equal margins when content/boxes are centered on the page. Don't want to extend the paragraph length. Most of the sections strike me as a bit two wide on a larger display so I'd be fine with reducing the overall margins of the page.

**Comment by aulneau** [comment]
> Think I mean the other sections of the page have equal margins when content/boxes are centered on the page. 

Yeah but the other sections aren't just copy.

If we were to restrict the max width of the text like we are so they don't get too long to read comfortably, and also center the content so it's not off to the left, this is what we get:

![Screen Shot 2019-01-09 at 4.56.30 PM.png](https://images.zenhubusercontent.com/5c0160f25cb43a45a18af171/2a5d5932-be98-43f6-b321-28b71eb038c4)

![Screen Shot 2019-01-09 at 4.56.35 PM.png](https://images.zenhubusercontent.com/5c0160f25cb43a45a18af171/e106941e-d373-468c-b75b-71c57ceb8e00)

Notice how it's no longer lining up on the left with the header or other elements, which is the reason I did not center the content.

The figma shows the text expanding to the width of the content, which is ~1000px which I think is too long. Ideally I think text should stay at a max width of around 768px, I've set it to 900px currently.
